### ISSUE TITLE
KeyError: 'pylint-per-file-ignores'

### ISSUE THREAD

**Issue by dscrofts** [christopherpickering/pylint-per-file-ignores#90]
## System Info

- OS: macOS 11.7.10
- pylint: 2.17.5
- pylint-per-file-ignores: 1.3.1

## Issue

With the latest version of `pylint-per-file-ignores`, a `KeyError` is raised when running `pylint`. 

## How To Reproduce

pyproject.toml:
```toml
[tool.pylint.master]
load-plugins=[
    "pylint_per_file_ignores",
]

[tool.pylint.messages_control]
per-file-ignores=[
    "test.py:C0116,E0001",
]
```

Run:
`pylint *.py`

Output:
```
Traceback (most recent call last):
  File "/usr/local/bin/pylint", line 8, in <module>
    sys.exit(run_pylint())
             ^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/pylint/__init__.py", line 36, in run_pylint
    PylintRun(argv or sys.argv[1:])
  File "/usr/local/lib/python3.11/site-packages/pylint/lint/run.py", line 169, in __init__
    args = _config_initialization(
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/pylint/config/config_initialization.py", line 113, in _config_initialization
    linter.load_plugin_configuration()
  File "/usr/local/lib/python3.11/site-packages/pylint/lint/pylinter.py", line 411, in load_plugin_configuration
    module_or_error.load_configuration(self)
  File "/usr/local/lib/python3.11/site-packages/pylint_per_file_ignores/__init__.py", line 268, in load_configuration
    ignores = {**content["tool"]["pylint-per-file-ignores"]}
                 ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyError: 'pylint-per-file-ignores'
```

**Comment by christopherpickering** [comment]
Thanks for reporting!
